README.md

```diff
@@ -2,40 +2,55 @@
 Let's make a planetary system!
 
 ## Learning Goals
-### Wave 1
-- Create a custom **class**
-- Create and use an **initialize** method in your class
-- Create and use **instance variables** in your class
 
-### Wave 2
-- Use an **array** to store a list of objects
-- Create and use a **method**
+- Practice creating Custom classes
+- Use a hash to initialize an object
+- Create an object which contains a collection of other objects
+- Practice creating instance methods
+
 
 ## Baseline
-- Create a `Planet` class with a name attribute.
-- You should be able to instantiate a new `Planet` object with an associated name.
+- Create a `SolarSystem` class with an `@planets` instance variable.
+- The initialize method should take a collection of planet names and store them in `@planets`
+- Create a method to add a planet to the list.
+- Create a method which will return a list of the planets in this style:
+
+```bash
+1.  Mercury
+2.  Venus
+3.  Earth
+4.  Mars
+5.  Jupiter
+6.  Hoth
+```
+
+
 
 # Wave 1
 ## Primary Requirements
-- Get creative! Give each instance of `Planet` at least five attributes. These could be diameters, mass, moons... whatever!
-- Allow these attributes to be set using a hash in `initialize`.
-- You should be able to create many different planets with different properties, like Mercury, Venus, Earth, Mars, Jupiter, etc.
+- Get creative! Instead of Strings for planets, give SolarSystem a list of hashes where each planet is sent as a hash with at least 5 attributes.  
 
 ## Optional Enhancements
 - Give each planet a rate of solar rotation
```

Comment:
Can we call this `year_length` or something like that? Lots of folks seemed confused by this verbiage. We'll have to change it below in the wave 3 optionals as well.